packages/docs/src/styles/components/responsiveExample.scss

```diff
@@ -0,0 +1,60 @@
+.c-resposive-example__example-wrapper {
+  margin: $spacer-2;
+}
+
+.c-repsonsive-example__iframe-wrapper {
+  transform-origin: 0px 0px;
+  overflow: hidden;
+}
+
+.c-responsive-example__iframe {
+  border: 0;
+  width: 100%;
+}
+
+.c-responsive-example__iframe-wrapper--width-xs {
+  width: 360px;
+}
+.c-responsive-example__iframe-wrapper--width-sm {
```

Comment:
These will eventually be changed to `$media-width-xx` I wonder if a TODO could be added or something to reference later when tokens go in.

Reply:
I suppose a search/replace later will bring this up, can disregard.